js/src/frontend/TokenStream.cpp

```diff
@@ -834,7 +834,7 @@ uint32_t TokenStreamAnyChars::computePartialColumn(
 #ifdef DEBUG
                                     this,
 #endif
-                                    start, limit,
+                                    start, limit, ColumnChunkLength,
```

Comment:
[nit] Maybe put it on its own line so we diff more cleanly.

Maybe add a `static` to the definition of `ColumnChunkLength` instead? It shouldn't be necessary to add it to the capture list of this lambda.

Reply:
Adding the `static` worked. Thanks!